lib/mayaUsd/ufe/UsdUndoSetKindCommand.h

```diff
@@ -26,6 +26,8 @@
 
 #include <memory>
 
+PXR_NAMESPACE_USING_DIRECTIVE
```

Comment:
Was relying on UsdUndoableItem.h to define this, but UsdUndoableItem.h doesn't need this definition and was removed.

Reply:
Hey @ppt-adsk! I noticed this pattern in some of the other UFE classes and I actually intentionally avoided doing the same here when I added this command in #1094. This macro effectively resolves to a `using namespace ...` statement which it seems like we would want to avoid in headers:
https://github.com/PixarAnimationStudios/USD/blob/dc710925675f7f58f7a37f6c18d046a687b09271/pxr/pxr.h.in#L52

Instead I had attempted to resolve the symbols by prefixing all of the USD types with `PXR_NS::`, but looks like I probably missed a few. Reading through that class again, I would guess it's at least these in the header:
https://github.com/Autodesk/maya-usd/blob/a9f81198c18d268038a7d9e0694fda4dc4ccdede/lib/mayaUsd/ufe/UsdUndoSetKindCommand.h#L55
https://github.com/Autodesk/maya-usd/blob/a9f81198c18d268038a7d9e0694fda4dc4ccdede/lib/mayaUsd/ufe/UsdUndoSetKindCommand.h#L57

And maybe this in the implementation:
https://github.com/Autodesk/maya-usd/blob/a9f81198c18d268038a7d9e0694fda4dc4ccdede/lib/mayaUsd/ufe/UsdUndoSetKindCommand.cpp#L54

Though I think using the `PXR_NAMESPACE_USING_DIRECTIVE` macro in a .cpp file would be fair game if we wanted to do that instead.

Reply:
Makes perfect sense, fully agree with you.  Let me wait until a bit more feedback builds up and I'll fix this.

Reply:
Sounds good, thanks! And sorry for leaving that trap for you. :)

Reply:
Are we ready to set a course on getting rid of all `using` directives from the header files in the core modules? That would be fantastic.